PRCS/prc-1.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Paima Achievement Interface
+description: Interface for sharing in-game achievements.
+author: seba@paimastudios.com, edward@paimastudios.com
```

Comment:
probably we should add Tad's name as well

Reply:
added